migration/src/main/java/uk/ac/bbsrc/tgac/miso/migration/destination/MigrationAuthorizationManager.java

```diff
@@ -83,4 +83,9 @@ public void throwIfNotWritable(SecurableByProfile resource) throws IOException,
     return new ArrayList<>(unfiltered);
   }
 
+  @Override
+  public void throwIfNonAdminOrMatchingOwner(User owner) throws IOException, AuthorizationException {
+
+  }
```

Comment:
Yes, I did look at the Javadoc; perhaps it's that I'm still learning the Java conventions as to when it's ok to leave methods blank.